src/syntax/ppx_bisect.ml

```diff
@@ -0,0 +1,13 @@
+open Migrate_parsetree
+open Ppx_tools_404
+
+let () =
+  Driver.register ~name:"ppx_bisect" ~args:InstrumentArgs.switches
+    Versions.ocaml_404 (fun _config _cookies ->
+        (* let anon s = raise (invalid_arg ("nothing anonymous: " ^ s)) in *)
+        let anon _s = () in
```

Comment:
Simply because I wasn't even able to run `./bisect_ppx --as-ppx` at all without removing this exception. I'll fix it up once I understand how to handle args correctly with omp